### PR TITLE
Create the right data structure to store data in memory

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Create the right data structure
* With proper usage of constructor

### Type of change
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Chore (a non-breaking change which doesn't affect existing functionality)

### Description of Task to be completed?

* Move data to constructor

### How should this be manually tested?

* git clone the repo
* npm install
* npm run dev

### What are the relevant pivotal tracker stories?

#164548977

### CODE REVIEW

File: server/services/messageServices.js [hunk 1]
```diff
@@ -175,35 +178,38 @@ export default class MessageService {
...
       });
     }
+    this.messages.push(newMessage);
     return newMessage;
   }
 
   getMessageById(id) {
-    const message = this.AllMessage()[id - 1];
+    const foundMessage = this.AllMessage().find(element => element.id === Number(id));
     if (!id) {
```
Comment (CEOehis) [inline server/services/messageServices.js:186]: This check should come before you use the `id` to look for the message in the list of messages.

File: server/services/messageServices.js [hunk 2]
```diff
@@ -175,35 +178,38 @@ export default class MessageService {
...
       return 'error';
     }
-    return message;
+    return foundMessage;
   }
 
   deleteMessage(id) {
-    const message = this.AllMessage()[id - 1];
+    const foundMessage = this.messages.find(element => element.id === Number(id));
     if (!id) {
```
Comment (CEOehis) [inline server/services/messageServices.js:197]: This check for the `id` should also come before you search for the message

**Review by CEOehis — CHANGES_REQUESTED** [review]
Also remove the build folder and add it to your `.gitignore`.

File: server/services/messageServices.js (hunk 1 above)
Comment (Tyak99) [reply server/services/messageServices.js:186]: Done

File: server/services/messageServices.js (hunk 2 above)
Comment (Tyak99) [reply server/services/messageServices.js:197]: Done

**Review by CEOehis — APPROVED** [review]
Looks good to me

**Pull request merged**